apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepConfirmation.tsx

```diff
@@ -1,8 +1,9 @@
+// FIXME: to update when implementing edit transaction on evm
+
 import React from "react";
```

Comment:
I think you should remove the following code in
/ledger-live-desktop/src/renderer/modals/Send/steps/StepRecipient.tsx
```
{stuckAccountAndOperation ? (
        <EditOperationPanel
          operation={stuckAccountAndOperation.operation}
          account={stuckAccountAndOperation.account}
          parentAccount={stuckAccountAndOperation.parentAccount}
        />
      ) : null}
```